Recast/iTunesPodcasts/Models/RSS/mapCharacters.swift

```diff
@@ -0,0 +1,114 @@
+    func map(_ string: String, for path: RSSPath) {
+        
+        switch path {
+        case .rssChannelTitle:                                      self.title                                                      = self.title?.appending(string) ?? string
+        case .rssChannelLink:                                       self.link                                                       = self.link?.appending(string) ?? string
+        case .rssChannelDescription:                                self.description                                                = self.description?.appending(string) ?? string
+        case .rssChannelLanguage:                                   self.language                                                   = self.language?.appending(string) ?? string
+        case .rssChannelCopyright:                                  self.copyright                                                  = self.copyright?.appending(string) ?? string
+        case .rssChannelManagingEditor:                             self.managingEditor                                             = self.managingEditor?.appending(string) ?? string
+        case .rssChannelWebMaster:                                  self.webMaster                                                  = self.webMaster?.appending(string) ?? string
+        case .rssChannelPubDate:                                    self.pubDate                                                    = string.toPermissiveDate()
+        case .rssChannelLastBuildDate:                              self.lastBuildDate                                              = string.toPermissiveDate()
+//        case .rssChannelCategory:                                   self.categories?.last?.value                                    = self.categories?.last?.value?.appending(string) ?? string
+        case .rssChannelCategory:                                   self.categories?
+            .append(string)
+        case .rssChannelGenerator:                                  self.generator                                                  = self.generator?.appending(string) ?? string
+        case .rssChannelDocs:                                       self.docs                                                       = self.docs?.appending(string) ?? string
+        case .rssChannelRating:                                     self.rating                                                     = self.rating?.appending(string) ?? string
+        case .rssChannelTTL:                                        self.ttl                                                        = Int(string)
+        case .rssChannelImageURL:                                   self.image                                                 = URL(string: string)
+//        case .rssChannelImageURL:                                   self.image?.url                                                 = self.image?.url?.appending(string) ?? string
```

Comment:
just a question... what's the difference between `image` and `image.url` in this case?

Reply:
Image in RSS can contain more subfields than just a URL, mainly for not being able to load the image (alt text for HTML). We don't really need that.